pkgs/by-name/ml/mlc/package.nix

```
@@ -19,19 +20,22 @@ stdenv.mkDerivation rec {
     install -Dm755 mlc $out/bin/mlc
   '';
 
-  nativeBuildInputs = [ patchelf ];
+  nativeBuildInputs = [ autoPatchelfHook ];
 
-  fixupPhase = ''
-    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" $out/bin/mlc
-  '';
+  passthru.tests.version = testers.testVersion {
+    package = finalAttrs.finalPackage;
+    command = "${finalAttrs.meta.mainProgram} --help";
+    version = "v${finalAttrs.version}";
+  };
 
-  meta = with lib; {
+  meta = with lib; rec {
```

```suggestion
  meta = rec {
```

Not completely on topic, but I'd remove the `with lib;` statement because of #208242 :)

---

I've made this change (and I appreciate you taking a look at this PR!) but I think it's worth noting that the [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/de578da2e1099044096a515ff2c5a4e5486ae291/CONTRIBUTING.md#syntax) discourage stylistic feedback for rules that are not yet incorporated into `CONTRIBUTING.md`:

> Any style choices not covered here but that can be expressed as general rules should be left at the discretion of the authors of changes and _not_ commented in reviews.